itests/src/test/java/fi/metatavu/edelphi/test/ui/base/index/ChangePlanTestsBase.java

```diff
@@ -0,0 +1,179 @@
+package fi.metatavu.edelphi.test.ui.base.index;
+
+import java.text.DateFormat;
+import java.text.NumberFormat;
+import java.time.OffsetDateTime;
+import java.util.Currency;
+import java.util.Date;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import fi.metatavu.edelphi.test.mock.PlanMocker;
+import fi.metatavu.edelphi.test.ui.base.AbstractUITest;
+
+public class ChangePlanTestsBase extends AbstractUITest {
+  
+  private static final String CHANGEPLAN_PAGE = "/changeplan.page";
+  
+  private PlanMocker planMocker = new PlanMocker();
+  
+  @Before
+  public void beforeTest() {
+    planMocker.createPlan("Plus 100", "Plus plan for 100 days with 100 euros", 100, "PLUS", "EUR", 100d, true);
+    planMocker.createPlan("Plus 200", "Plus plan for 200 days with 180 euros", 200, "PLUS", "EUR", 180d, true);
+    planMocker.createPlan("Premium 100", "Premium plan for 100 days with 200 euros", 100, "PREMIUM", "EUR", 200d, true);
+    planMocker.createPlan("Premium 200", "Premium plan for 200 days with 360 euros", 200, "PREMIUM", "EUR", 360d, true);
+    planMocker.mock();
+  }
+  
+  @After
+  public void afterTest() {
+   deleteOrderHistories();
+    planMocker.cleanup();
+  }
+  
+  @Test
+  public void testBasicToPlus() {
+    testPlanChange(null, "BASIC", "Basic", null, 1, 100, "PLUS", "Plus", 100, null, null);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "BASIC" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Basic" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)